auth/src/test/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivityTest.java

```diff
@@ -171,32 +171,27 @@ public void testGoogleLoginFlow() {
 
         assertNotNull(googleButton);
         googleButton.performClick();
-        ShadowActivity.IntentForResult nextIntent =
-                Shadows.shadowOf(authMethodPickerActivity).getNextStartedActivityForResult();
 
-        verifySaveCredentialIntent(nextIntent, GoogleAuthProvider.PROVIDER_ID);
+        assertEquals(
+                Shadows.shadowOf(authMethodPickerActivity).getResultCode(),
+                Activity.RESULT_OK);
```

Comment:
If I could get robolectric to wait for `SmartLock` to finish, `Shadows.shadowOf(authMethodPickerActivity).getResultCode()` would change from `Activity.RESULT_CANCELED` to `Activity.RESULT_OK` which would confirm that the something happened if not the credentials being saved. Any ideas?